__tests__/app.test.js

```diff
@@ -83,6 +83,34 @@ describe("GET Requests", () => {
         });
     });
   });
+  describe("/api/articles", () => {
+    test("status 200: responds with an array of article objects, all including comment_count. articles sorted by date in desc. order", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+
+          expect(articles).toBeInstanceOf(Array);
+          expect(articles).toHaveLength(12);
+          expect(articles).toBeSortedBy("created_at", { descending: true });
+          articles.forEach((article) => {
+            expect(article).toEqual(
+              expect.objectContaining({
+                author: expect.any(String),
+                title: expect.any(String),
+                article_id: expect.any(Number),
+                body: expect.any(String),
```

Comment:
The ticket does not ask for the body to be included in the returned article object